angular5/app/app.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core'
+import { BrowserModule } from '@angular/platform-browser'
+import { AppComponent } from './app.component'
+
+import jss from 'jss'
+import preset from 'jss-preset-default'
+
+@NgModule({
+  imports: [BrowserModule],
+  declarations: [AppComponent],
+  bootstrap: [AppComponent]
+})
+export class AppModule {
+  constructor() {
+    jss.setup(preset())

```

Comment:
better to do this statically, outside of a constructor, because it needs to be done only once